docker-compose.yml

```diff
@@ -48,6 +48,6 @@ services:
               read_only: true
             - type: bind
               source: ${TESTSET_DATA_FOLDER}/database_config.yaml
-              target: /app/database_config.yaml
+              target: /preprocessing/input/database_config.yaml
```

Comment:
This is actually not necessary anymore as it is already mounted with the entire folder